.circleci/config.yml

```diff
@@ -0,0 +1,262 @@
+          paths:
+            - .\python
+
+      # Set up virtual environment
+      - run:
+          name: create virtualenv
+          command: |
+            .\python\tools\python.exe --version
+            .\python\tools\python.exe -m pip install virtualenv
+            .\python\tools\python.exe -m virtualenv env
+      # Install requirements
+      - restore_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+      - run: &win-install-dependencies-template
+          name: install dependencies
+          command: |
+            env\Scripts\activate.ps1
+            python --version
+            pip install -r requirements.txt
+            pip install coverage
+      - save_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+          paths:
+            - env
+
+      # Run tests
+      - run: &win-run-unittests-template
+          name: run unittests
+          command: |
+            env\Scripts\activate.ps1
+            coverage run -m unittest discover
+  test-win-py37:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.7.5
+
+  test-win-py36:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.6.8
+
+  test-win-py35:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.5.4
+
+  #
+  # Windows x86
+  #
+
+  test-win-py38x86: &win-tests-template-x86
+    executor:
+      name: win/default
+
+    environment:
+      PYTHON: 3.8.0
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Install python
+      - restore_cache:
+          key: nuget-{{ .Environment.CIRCLE_JOB }}
+      - run:
+          name: install python
+          command: nuget install pythonx86 -Version $env:PYTHON -ExcludeVersion -OutputDirectory .
+      - save_cache:
+          key: nuget-{{ .Environment.CIRCLE_JOB }}
+          paths:
+            - .\python
+
+      # Set up virtual environment
+      - run:
+          name: create virtualenv
+          command: |
+            .\pythonx86\tools\python.exe --version
+            .\pythonx86\tools\python.exe -m pip install virtualenv
+            .\pythonx86\tools\python.exe -m virtualenv env
+      # Install requirements
+      - restore_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+      - run: *win-install-dependencies-template
+      - save_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+          paths:
+            - env
+
+      # Run tests
+      - run: *win-run-unittests-template
+
+  test-win-py37x86:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.7.5
+
+  test-win-py36x86:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.6.8
+
+  test-win-py35x86:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.5.4
+
+
+# TODO: add Py3.8 in jobs once ortools (for penaltymodel-mip) supports Py3.8
```

Comment:
When you fix the above issues, all x86 tests in windows should fail. It's not just about ortools on py38, but about 32-bit python on 64-bit win nodes on CircleCI.

Check https://github.com/dwave-examples/qboost/pull/6 to see how my attempt to "fix py38 support on windows" ended and [this commit message](https://github.com/dwave-examples/qboost/pull/6/commits/512c43a5dcc1d6e9ef0c0efd0c1cd2f1d837b436) for gory details.

Reply:
TL;DR: just drop the win x86 tests.

Reply:
https://github.com/dwavesystems/olp-content/pull/138#pullrequestreview-454413729